### DISCUSSION TITLE
Mistakes (typos) in the whole series

### DISCUSSION THREAD

**Discussion by apoapsis16** [DDReaper/XNAGameStudio#20] [General]
Found a couple really confusing typos on here, for example in "Riemers3DXNA1Terrain07terrainbasics"

"To begin with, we will not be using the Z coordinate so that you can see the difference later on in this chapter."

We are not using the Y (height) coordinate not Z.

and then later;

"These will require 32 * 3 = 18 indices (=(terrainWidth-1)(terrainHeight-1)*3)*."

This confusing mess should read:

These will require 6 * 3 = 18 indices =(terrainWidth-1) * ((terrainHeight-1)*3).

**Comment by SimonDarksideJ** [comment]
Thanks for the feedback, likely just errors in the migration, as predominately the text did not change from the original.

PR's are welcome too btw :D 

Will get on this asap.

**Comment by apoapsis16** [comment]
What is a PR? :)

**Comment by SimonDarksideJ** [comment]
![giphy](https://github.com/SimonDarksideJ/XNAGameStudio/assets/1793042/dbf0d253-a9af-4469-bcd1-a547f117e1f3)

**Comment by apoapsis16** [comment]
Oh a Pull Request? It was a genuine question lol. I have a fix for another issue with the HLSL tutorial, can I make a PR for a wiki article tho? Anyway I really appreciate you putting these tutorials up and porting them. Thanks so much

**Reply by SimonDarksideJ** [reply]
Apologies, I see what you mean. I Forgot GitHub doesn't really do PR's to Wiki pages.

I'm going to setup some time to convert the Wiki to a GitHub pages site (the wiki will have to remain due to SEO), but it should makes the docs side a LOT more maintainable in the future.

If you're serious, then I would be happy to give you `Collaborator` status, as it is a lot of effort to maintain this site (while also maintaining the official MonoGame docs :D )

**Reply by apoapsis16** [reply]
Sure thing. When I see the github pages I'll add the PR with the fix for the HLSL project

**Comment by SimonDarksideJ** [comment]
@apoapsis16 , the GitHub pages site is not up and running :D

https://darkgenesis.zenithmoon.com/XNAGameStudio.github.io/

Any help you can offer to to updates and suggest content would be most welcome. SImply:

- Fork the repo
- make changes in a NEW branch in your fork
- Make a PR back to the base repo

Really glad for the help as it is a lot of work to maintain (and upgrade :D )

The site is on it's "default" link of (https://darkgenesis.zenithmoon.com/XNAGameStudio.github.io/) atm, but I'll put a custom URL in when it's ready and then make modifications to the original Wiki to redirect people there.
